GoogleUtilities/Tests/Unit/Environment/GULKeychainStorageTests.m

```diff
@@ -87,6 +87,8 @@ - (void)testSetGetObjectForKey {
 
   // 3. Read existing object which is not present in in-memory cache.
   [self.cache removeAllObjects];
+  // TODO: Evaluate if GULKeychainStorage needs an API that takes set of classes. (#42)
+  // The following method causes an NSKeyedUnarchiver-related runtime warning log.
   [self assertSuccessReadObject:@{@"key" : @"value"}
                          forKey:@"test-key1"
                           class:[NSDictionary class]
```

Comment:
This method calls `GULKeychainStorage`'s public API [`getObjectForKey:(NSString *)key objectClass:(Class)objectClass`](https://github.com/google/GoogleUtilities/blob/main/GoogleUtilities/Environment/SecureStorage/GULKeychainStorage.m#L60) and logs a runtime `NSKeyedArchiver` warning because the `NSString` class needs to be specified as well. 

Currently, there is not an API for `GULKeychainStorage` that takes a set of classes but we might consider adding one. A quick code search shows the `getObjectForKey: ...` API is only used in [Installations](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseInstallations/Source/Library/InstallationsStore/FIRInstallationsStore.m#L63) and that use doesn't result in a warning (because a set of classes is not needed there).

Since this warning is only logged in tests, I've added a TODO for now.